app/assets/javascripts/poll_feeds.js

```diff
@@ -0,0 +1,63 @@
+var spinnerIndex = -1;
```

Comment:
This is our javascript file that makes ajax calls to poll the status of the `PullFeedsJob` sidekiq job that loads our feeds.